[Bug] deploy.yml에서 docker-compose → docker compose로 수정 필요

## 📌 개요
Self-hosted Runner 기반 CI/CD 배포에서 `docker-compose` 명령어를 사용해 배포하도록 작성했으나,  
Runner 환경에는 Docker Compose v2(`docker compose`)만 설치되어 있어 오류가 발생함.

## 🐞 문제 상황
- 배포 워크플로우 실행 시 에러 발생:
- Error: Process completed with exit code 127.
  - 127 = “command not found”
- 원인: RPi 환경에는 `docker compose` (Compose v2)만 설치됨
- 결과: `docker-compose` 명령어 실행 불가 → 배포 실패

## ✅ 해결 방안
- `.github/workflows/deploy.yml` 내 모든 `docker-compose` 명령어를 `docker compose`로 교체
- 불필요한 패키지 설치 단계(`apt-get install docker-compose`) 제거

## 🔗 참고
- Issue #17 (Runner 기반 배포 PR 이후 발견된 문제)
- GitHub Actions 실행 로그 캡처

<img width="1865" height="1262" alt="Image" src="https://github.com/user-attachments/assets/97286368-2906-48f8-be3f-f23215105e90" />

--- a/.github/workflows/deploy.yml
+++ b/.github/workflows/deploy.yml
@@ -26,9 +26,9 @@ jobs:
             git reset --hard origin/main
 
             echo "=== Docker 이미지 Pull ==="
-            docker-compose -f services/docker-compose.yml pull
+            docker compose -f services/docker-compose.yml pull
 
-            echo "=== 서비스 재시작 (변경만 반영) ==="
-            docker-compose -f services/docker-compose.yml up -d --build
+            echo "=== 서비스 재시작 ==="
+            docker compose -f services/docker-compose.yml up -d --build
 
             echo "=== 배포 완료 ==="

